build.gradle

```diff
@@ -110,6 +110,7 @@ subprojects { project ->
     jvmArgs += "-Dokhttp.platform=$platform"
 
     testLogging {
+//      events "skipped", "failed"
```

Comment:
?

Reply:
It's useful feedback when debugging in CI, so keeping here to remember the code, I think instead I need to work out how to control the logging levels of the gradlew.

Reply:
if (false) ? That way find-usages etc. work

Reply:
Never comment out code, but `if (false)` is completely fine ’cause it can be managed through refactoring and other static tools